src/components/inputs/InputField.vue

```diff
@@ -124,12 +125,24 @@ export default defineComponent({
         );
 
         const visible = computed(
-            () =>
-                !props.input.showIfAll ||
-                props.input.showIfAll.every(
+            () => {
+                if (props.input.showIf != undefined) {
```

Comment:
I think testing for undefined is too strict (and in any case should be done with `!==` to ensure it's actually `undefined` and not `null`). If the value is `null` we don't want to check it either, and JsonLogic has to be an object, so it feels fine.
```suggestion
                if (props.input.showIf) {
```

Reply:
I inherently get the heebie-jeebies by trueish values, but I guess it's more idiomatic in js. Thanks!